packages/astro-rss/README.md

```diff
@@ -196,59 +57,59 @@ const item = {
 };
 ```
 
-### `title`
+#### `title`
```

Comment:
Hm, adding a `### Properties` wouldn't be accurate. These are properties for the `items` array, so it would become `#### Properties` and _level 5 headings_ for each property 😬 

We originally broke out the sub properties to a separate section, but I found that to be confusing. Thoughts on some sort of lead-in sentence to clarify the nested headings?

Reply:
Ah, I should have expanded the hidden text for more context! I have *some* ideas, but I think it's probably fine as is!